### DISCUSSION TITLE
Preflight request ?

### DISCUSSION THREAD

**Discussion by PedroFonsecaDEV** [artilleryio/artillery#1702] [Artillery]
When firing a  POST request using Artillery, does Artillery will fire a pre-flight request? If not, where I can find an example of how to simulate a pre-flight request (method: OPTIONS)?

**Comment by hassy** [comment]
hi @PedroFonsecaDEV 👋 

Artillery does not issue pre-flight requests automatically, but you can send an `OPTIONS` request like you would any other type of request. It's possible to capture parts of the response etc as normal as well.

Quick example here: https://play.artillery.io/?s=232a83c0-4fae-41e0-91b8-db573624e8c4. The output on the right contains the values of captured headers returned for the `OPTIONS` request.

(Moving this to Discussions)

**Reply by hassy** [reply]
Just noticed we don't mention `OPTIONS` requests in the docs, will fix!
